b2sdk/api.py

```diff
@@ -386,6 +388,7 @@ def list_buckets(self, bucket_name=None, bucket_id=None):
 
         :param str bucket_name: the name of the one bucket to return
         :param str bucket_id: the ID of the one bucket to return
+        :param bool cache: if ``True`` use cached bucket list if available
```

Comment:
```suggestion
        :param bool use_cache: if ``True`` use bucket list cached in account_info if available and non-empty
```

Reply:
will rename param, but I will skip "in account_info" text we have other cache implementations as well, so this is not necessary always true